Xamarin.Forms.Platform.Android/Cells/ViewCellRenderer.cs

```diff
@@ -5,11 +5,14 @@
 using System;
 using System.Linq;
 using Android.Runtime;
+using System.ComponentModel;
 
 namespace Xamarin.Forms.Platform.Android
 {
 	public class ViewCellRenderer : CellRenderer
 	{
+		ViewCellContainer _view;
```

Comment:
I think this need change type to `WeakReference<ViewCellContainer>`. To avoid memory leak.

Reply:
Should be fixed now